docs/documentation/upgrading/topics/keycloak/changes-24_0_0.adoc

```diff
@@ -26,6 +26,38 @@ Images such as the background, logo and favicon are now loaded from the 'common'
 import=common/your-theme-name
 ----
 
+= Changes to the Account Console theme customization
+
+If you were previously extending the now deprecated version 2 of the Account Console theme, you will need to update your theme to use the new version 3 of the Account Console theme. The new version of the Account Console theme comes with some changes in regards to how it is customized. To start with a clean slate you can follow the new https://github.com/keycloak/keycloak-quickstarts/tree/latest/extension/extend-account-console[customization quickstart].
+
+To move your custom theme start by changing the `parent` to the new theme:
+
+[source,properties]
+----
+# Before
+parent=keycloak.v2
+
+# After
+parent=keycloak.v3
+----
+
+If you have any custom React components, you will have to import React directly, rather than using a relative path:
```

Comment:
```suggestion
If you have any custom React components, you will import React directly, rather than using a relative path:
```

Reply:
We avoid "you must" and "you have to" which sounds dictatorial.  Removing "must" or "have" loses no meaning. It is more direct and simple.